iqm5q.yml

```diff
@@ -19,233 +19,188 @@ native_gates:
-    #                     amplitude: 0.5
-    #                     frequency: 0
-    #                     shape: RTZ()
-    #                     phase: null
-    #                     channel: 2
-    #                     type: qf # qubit flux
-    #                 -   start: 0
-    #                     duration: 4
-    #                     amplitude: 0.01
-    #                     frequency: 0
-    #                     shape: RTZ()
-    #                     phase: null
-    #                     channel: 2
-    #                     type: qf # qubit flux
 characterization:
     single_qubit:
         0:
-            readout_frequency: 5_229_200_000 #5_233_200_000
+            readout_frequency: 5_228_700_000
             resonator_polycoef_flux: []
-            drive_frequency: 4_095_830_788 #4_093_830_788
+            drive_frequency: 4095985280
             T1: 0.0
             T2: 0.0
-            sweetspot: 0.05 #Use this as an amplitude of the flux_bias
+            sweetspot: 0
             filter: {}
-            mean_gnd_states: 1.5420+0.1813j
-            mean_exc_states: 2.4499-0.5629j
+            mean_gnd_states: 1.5417+0.1817j
+            mean_exc_states: 2.5332-0.5914j
             # parameters for single shot classification
-            threshold: 0.8836
-            iq_angle: -1.551
-            # alpha: 217.492 MHz
-            # To save power values on the runcard
-            # ro_range_lp: -15
-            # ro_range_hp: -15
-            # qd_range: 0
-            # flux_range: -0
-
+            threshold: 1.5939
+            iq_angle: 0.662
+            alpha: 217_492_000
         1:
-            readout_frequency: 4_931_000_000
+            readout_frequency: 4.9525e+9
             resonator_polycoef_flux: []
-            drive_frequency: 4_170_000_000
-            T1: 0.
-            T2: 0.
-            sweetspot: 0.0
+            drive_frequency: 4.25e+9
+            T1: 0.0
+            T2: 0.0
+            sweetspot: 0
             filter: {}
             mean_gnd_states: (0+0j)
             mean_exc_states: (0+0j)
         2:
-            readout_frequency: 6.109e+9  #6_112_000_000
+            readout_frequency: 6.109e+9
             resonator_polycoef_flux: []
-            drive_frequency: 4_300_587_281 # 4_401_600_000 #4_541_100_000
-            T1: 0.
-            T2: 0.
-            sweetspot: 0.0 #0.5 #0.62 0.00
+            drive_frequency: 4342116873
+            T1: 0.0
+            T2: 0.0
+            sweetspot: -0.1745
             filter: {}
-            mean_gnd_states: -1.8243+1.5926j
-            mean_exc_states: -0.8083+2.3929j
+            mean_gnd_states: -1.8517+0.6431j
+            mean_exc_states: -0.8696+1.1473j
             # parameters for single shot classification
-            threshold: -0.0593
-            iq_angle: -0.667
-            # alpha: 208 MHz
+            threshold: -0.6085
+            iq_angle: -0.474
+            alpha: 211_604_296
```

Comment:
Is this `alpha` properly loaded in the platform/qubits? Because if it is not a field of the `Qubit` dataclass in qibolab, I am not sure if it will be properly loaded.